InventoryWatch/MenuBarView.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  MenuBarView.swift
+//  InventoryWatch
+//
+//  Created by Ramik Sadana on 09/28/22.
+//
+
+import SwiftUI
+
+struct MenuBarView: View {
+    @EnvironmentObject var model: Model
+    
+    @AppStorage("lastUpdateDate") private var lastUpdateDate: String = ""
+    
+    var body: some View {
+        ScrollView(.vertical) {
+            VStack(alignment: .leading, spacing: 5) {
```

Comment:
just curious - why a `VStack` (that contains another `VStack`) here instead of a `List`?

Reply:
SwiftUI `List` insists a very specific visual styling that I didn't think looked very good. Again, feel free to change this with the `DesignSystem` change.

The inner `VStack` is so that we don't run out of the 10 children limit of VStacks.